templates/map.html.twig

```diff
@@ -1,6 +1,12 @@
-{% set url = mapboxEncode(conferenceCoordinates) %}
+{% if conferenceCoordinates|length == 1 %}
+    {% set url = mapboxEncode(conferenceCoordinates, 5) %}
+{% else %}
+    {% set url = mapboxEncode(conferenceCoordinates) %}
+{% endif %}
```

Comment:
This section could be simplified: the zoom "guess" could be done in the twig function